src/interface/viewport.c

```diff
@@ -920,10 +927,12 @@ int sub_98196C(int image_id, sint8 x_offset, sint8 y_offset, sint16 length_x, si
  * @param z_offset (ah)
  * @param height (edx)
  * @param rotation
- * @return ??
+ * @param word_52 (0x009DEA52)
+ * @param word_54 (0x009DEA54)
+ * @param word_56 (0x009DEA56)
  */
-int sub_98197C(int image_id, sint8 x_offset, sint8 y_offset, sint16 length_x, sint16 length_y, sint8 z_offset, int height, uint32 rotation){
-	int ebp = z_offset + RCT2_GLOBAL(0x9DEA56, sint16);
+int sub_98197C(int image_id, sint8 x_offset, sint8 y_offset, sint16 length_x, sint16 length_y, sint8 z_offset, int height, uint32 rotation, sint16 s_unk_x, sint16 s_unk_y, sint16 word_56) {
```

Comment:
@IntelOrca @duncanspumpkin Pretty much all assignments are done with `uint16`, but reading is being done with `sint16`. Which one should it be?